start-tests.sh

```diff
@@ -35,26 +35,30 @@ fi
 
 # Load test config mounted at /resources/options.yaml
 OPTIONS_FILE=/resources/options.yaml
-USER_OPTIONS_FILE=./options.yaml
+USER_OPTIONS_FILE=./resources/options.yaml
```

Comment:
These files are different on purpose. First we try to read from `/resources/options.yaml` if the file doesn't exist then we read from `./options.yaml`

Reply:
Reverted user `options.yaml` path to its original path.